frontend/src/pages/LoginPage.js

```diff
@@ -49,38 +49,22 @@ const LoginPage = () => {
 
   const setSuccessDialog = (username) => {
     setIsDialogOpen(true)
-    setDialogTitle('Log in Success')
-    setDialogMsg(
-      <>
-        <DialogContentText>
-          {'You have successfully logged in.'}
-        </DialogContentText>
-        <DialogContentText>{`Welcome back, ${username}!`}</DialogContentText>
-      </>
-    )
+    setDialogTitle('Logged In Successfully')
+    setDialogMsg(`Hi, ${username}!👋 Welcome back to PeerPrep!`)
   }
 
   const setErrorDialog = (msg) => {
     setIsDialogOpen(true)
     setDialogTitle('Unable To Log In')
-    setDialogMsg(
-      <>
-        <DialogContentText>{msg}</DialogContentText>
-      </>
-    )
+    setDialogMsg(msg)
   }
 
   const redirectButton = () => {
     const redirectUrl = window.localStorage.getItem(AUTH_REDIRECT) ?? homeUrl
     // reset the redirect url since it's only used this one time
     window.localStorage.removeItem(AUTH_REDIRECT)
     return (
-      <Button
-        className="font-inter"
```

Comment:
I was trying and realised that it is not possible to use font-inter from Tailwind CSS to change the style of buttons in Dialog. Hence, I removed it.